packages/block/src/from-rpc.ts

```diff
@@ -1,35 +1,30 @@
 import { TransactionFactory } from '@ethereumjs/tx'
-import { setLengthLeft, toBuffer } from '@ethereumjs/util'
+import { TypeOutput, setLengthLeft, toBuffer, toType } from '@ethereumjs/util'
 
 import { blockHeaderFromRpc } from './header-from-rpc'
-import { numberToHex } from './helpers'
 
 import { Block } from './index'
 
 import type { BlockOptions, JsonRpcBlock } from './index'
 import type { TxData, TypedTransaction } from '@ethereumjs/tx'
 
-function normalizeTxParams(_txParams: any) {
+export function normalizeTxParams(_txParams: any) {
```

Comment:
Is it really necessary to export this function (always problematic from an API point of view, since the "we cannot change surface" will grow)? I am not seeing any external usage.

Reply:
I used it in the Ethers State Manager and I think it just got carried over when I copied this code from that PR.  I think we could remove the export for now and add it back later if/when we proceed with an Ethers State Manager PR that would get merged.

Reply:
Hmm, I guess from an API point-of-view this is not an optimal method to export regarding method location (block library), purpose, typing, docs....

So I would really like to remove again, in doubt just copy over the code to `EthersStateManager`. Or alternatively, if you have a solid and generic way to implement and document you can also move over to `Transaction` (I think this would make more sense in case we do?), but then we also need some tests and the whole way down. 🙂